dash/dash.py

```diff
@@ -378,15 +379,25 @@ def _relative_url_path(relative_package_path='', namespace=''):
 
         srcs = []
         for resource in resources:
+            is_dynamic_resource = 'dynamic' in resource and \
```

Comment:
perhaps `resource.get('dynamic', False)` so it is not split

Reply:
dictionary entry with default I guess?